lib/screens/login/login.dart

```diff
@@ -0,0 +1,109 @@
+import 'package:ecellapp/screens/login/cubit/login_cubit.dart';
+import 'package:ecellapp/screens/login/widgets/email_field.dart';
+import 'package:ecellapp/screens/login/widgets/password_field.dart';
+import 'package:ecellapp/screens/signup/signup.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class LoginScreen extends StatelessWidget {
+  final TextEditingController passwordController = TextEditingController();
+  final TextEditingController emailController = TextEditingController();
+  final _formKey = GlobalKey<FormState>();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: BlocConsumer<LoginCubit, LoginState>(
+        listener: (context, state) async {
+          if (state is LoginSuccess) {
+            await Future.delayed(Duration(seconds: 1));
+            Scaffold.of(context).showSnackBar(SnackBar(content: Text("Login Sucess")));
+          } else if (state is LoginError) {
+            Scaffold.of(context).showSnackBar(SnackBar(content: Text(state.message)));
+          }
+        },
+        builder: (context, state) {
+          if (state is LoginInitial) {
+            return _buildInitial(context);
+          } else if (state is LoginLoading) {
+            return _buildLoading();
+          } else if (state is LoginSuccess) {
+            return _buildSuccess();
+          } else if (state is SignUpRedirect) {
+            return SignupScreen();
+          } else if (state is ForgotPasswordRedirect) {
+            return Text("Replace With Screen when build!");
```

Comment:
Should be handled by listener. Don't do it now.
For future reference, never print a comment for the developers on the screen. Add a `// TODO` comment instead